site/themes/code-troopers/layouts/partials/nav.html

```diff
@@ -18,4 +18,5 @@
             {{ end}}
         {{ end }}
     </nav>
+    <div class="hiring"><a href="https://team.code-troopers.com/"><img src="img/hiring.png" title="On recrute">On recrute</a></div>
```

Comment:
What does it look like on mobile devices ?

Reply:
<img width="446" alt="Screenshot 2021-06-23 at 10 03 36" src="https://user-images.githubusercontent.com/1065693/123059797-6db0d900-d40a-11eb-9b92-ae75be1797e3.png">